Source/WebCore/PAL/pal/spi/cocoa/AVFoundationSPI.h

```diff
@@ -486,8 +486,7 @@ NS_ASSUME_NONNULL_END
 @end
 #endif
 
-// FIXME: Move into !USE(APPLE_INTERNAL_SDK) section once rdar://111695863 has been in the build a while
-#if HAVE(AVURLASSET_ISPLAYABLEEXTENDEDMIMETYPEWITHOPTIONS)
+#if !USE(APPLE_INTERNAL_SDK)
```

Comment:
Does this file not already have such a section to move these to?

Reply:
It does not appear to have a clear section, no. Lots of small islands.